qiskit_nature/circuit/library/initial_states/fermionic_gaussian_state.py

```diff
@@ -0,0 +1,121 @@
+           \end{pmatrix}
+           = W
+           \begin{pmatrix}
+                a^\dagger_1 \\
+                \vdots \\
+                a^\dagger_N \\
+                a_1 \\
+                \vdots \\
+                a_N
+           \end{pmatrix},
+
+        - :math:`a^\dagger_1, \ldots, a^\dagger_{N}` are the fermionic creation operators
+        - :math:`W` is an :math:`N \times 2N` matrix such that
+          :math:`b^\dagger_1, \ldots, b^\dagger_{N}` also satisfy the
+          fermionic anticommutation relations
+        - :math:`\lvert \overline{\text{vac}} \rangle` is the mutual 0-eigenvector of
+          the operators :math:`\{b_j^\dagger b_j\}`
+
+        The matrix :math:`W` has the block form
+
+        .. math::
+           \begin{pmatrix}
+                W_1 & W_2
+           \end{pmatrix}
+
+        where :math:`W_1` and :math:`W_2` must satisfy
+
+        .. math::
+            W_1 W_1^\dagger + W_2 W_2^\dagger = I \\
+            W_1 W_2^T + W_2 W_1^T = 0
+
+        Currently, only the Jordan-Wigner Transformation is supported.
+
+        Reference: arXiv:1711.05395
+
+        Args:
+            transformation_matrix: The matrix :math:`W` that specifies the coefficients of the
+                new creation operators in terms of the original creation and annihilation operators.
+                This matrix must satisfy special constraints; see the docstring of this function.
+            qubit_converter: a QubitConverter instance.
+            circuit_kwargs: Keyword arguments to pass to the QuantumCircuit initializer.
+
+        Raises:
+            ValueError: transformation_matrix must be a 2-dimensional array.
+            ValueError: transformation_matrix must have shape (n_orbitals, 2 * n_orbitals).
+            NotImplementedError: Currently, only the Jordan-Wigner Transform is supported.
+        """
+        if not len(transformation_matrix.shape) == 2:
+            raise ValueError("transformation_matrix must be a 2-dimensional array.")
+
+        n, p = transformation_matrix.shape  # pylint: disable=invalid-name
+        if p != n * 2:
+            raise ValueError("transformation_matrix must have shape (n_orbitals, 2 * n_orbitals).")
+        # TODO maybe check known matrix constraints
+
+        if occupied_orbitals is None:
+            occupied_orbitals = []
+        if qubit_converter is None:
+            qubit_converter = QubitConverter(JordanWignerMapper())
+
+        register = QuantumRegister(n)
+        super().__init__(register, **circuit_kwargs)
+
+        if isinstance(qubit_converter.mapper, JordanWignerMapper):
+            operations = _prepare_fermionic_gaussian_state_jordan_wigner(
+                register, transformation_matrix, occupied_orbitals
+            )
+            for gate, qubits in operations:
+                self.append(gate, qubits)
+        else:
+            raise NotImplementedError("Currently, only the Jordan-Wigner Transform is supported.")
```

Comment:
This should be more explicit in telling the user how to fix this (i.e. provide a `QubitConverter(JordanWignerMapper())`)

Reply:
Done.